container-gcp-go/main.go

```diff
@@ -97,7 +97,7 @@ func main() {
 				Spec: cloudrun.ServiceTemplateSpecArgs{
 					Containers: cloudrun.ServiceTemplateSpecContainerArray{
 						cloudrun.ServiceTemplateSpecContainerArgs{
-							Image: image.ImageName,
+							Image: image.RepoDigest,
```

Comment:
When I run this, I get:

```
Diagnostics:
  pulumi:pulumi:Stack (cloudrun-test-dev):
    error: error in compiling Go: unable to run `go build`: exit status 1

    # cloudrun-test
    ./main.go:100:15: cannot use image.RepoDigest (variable of type pulumi.StringPtrOutput) as pulumi.StringInput value in struct literal: pulumi.StringPtrOutput does not implement pulumi.StringInput (missing method ToStringOutput)
```

I believe it should be:

```suggestion
							Image: image.RepoDigest.Elem(),
```

Look okay?

Reply:
Went ahead and applied this so we could get it merged. Thank you!